requirements.txt

```diff
@@ -1,9 +1,9 @@
 certifi==2018.8.24
 chardet==3.0.4
-gspread==3.0.1
+gspread==0.6.2
 httplib2==0.11.3
 idna==2.7
-oauth2client==4.1.3
+oauth2client==4.1.2
```

Comment:
These to fixed versions should be in requirements/base.txt

Reply:
done